patches/k6-micro.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/output/csv/output.go b/output/csv/output.go
+index 582ad7e5..c4e94ba2 100644
+--- a/output/csv/output.go
++++ b/output/csv/output.go
+@@ -198,7 +198,7 @@ func SampleToRow(sample *metrics.Sample, resTags []string, ignoredTags []string,
+ 	case TimeFormatRFC3339:
+ 		row[1] = sample.Time.Format(time.RFC3339)
+ 	case TimeFormatUnix:
+-		row[1] = strconv.FormatInt(sample.Time.Unix(), 10)
++		row[1] = strconv.FormatInt(sample.Time.UnixMicro(), 10)
```

Comment:
Is this to get more precision out of the k6 output? I would have thought they'd have a high precision format by default?

Reply:
This is, for the CSV raw output they only output second-resolution timestamps without this change.

I don't think the `timePrecision` part of this comment has been added yet: https://github.com/grafana/k6/pull/2274#issuecomment-1085781856